modules/custom/activity_send/modules/activity_send_email/src/Plugin/ActivityDestination/EmailActivityDestination.php

```diff
@@ -40,4 +41,24 @@ public static function setSendEmailUserSettings($account, $values) {
     parent::setSendUserSettings('email', $account, $values);
   }
 
+  /**
+   * Get field value for 'output_text' field from data array.
+   */
+  public static function getSendEmailOutputText(Message $message) {
+    $value = NULL;
+    if (isset($message)) {
+      $value = $message->getText(NULL);
+      // Text for email.
+      if (!empty($value[2])) {
```

Comment:
I'm not sure if it will be clear to everyone that the 3rd value is used for e-mail templates. This might be something to improve upon later, or add description in form_alter in message_template form.